src/components/sessions/Session.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import PropTypes from 'prop-types'
+import { Button, Grid, Segment } from 'semantic-ui-react'
+import moment from 'moment'
+import QuestionInSession from './QuestionInSession'
+
+import withCSS from '../../lib/withCSS'
+
+const Session = ({ createdAt, head, id, intl, name, questions, status }) => {
+  let buttonContent = ''
+  let buttonIcon = ''
+  switch (status) {
+    case 'CREATED': {
+      buttonContent = intl.formatMessage({
+        id: 'session.button.created.content',
+        defaultMessage: 'Starten',
+      })
+      buttonIcon = 'play'
+      break
+    }
+    case 'RUNNING': {
+      buttonContent = intl.formatMessage({
+        id: 'session.button.running.content',
+        defaultMessage: 'Was hier?',
+      })
+      buttonIcon = 'play' // TODO which icon here?
+      break
+    }
+    case 'COMPLETED': {
+      buttonContent = intl.formatMessage({
+        id: 'session.button.completed.content',
+        defaultMessage: 'Kopieren',
+      })
+      buttonIcon = 'copy'
+      break
+    }
+    default: {
+      buttonContent = intl.formatMessage({
+        id: 'session.button.created.content',
+        defaultMessage: 'Starten',
+      })
+      buttonIcon = 'play'
+      break
+    }
+  }
+
+  return (
+    <Grid padded stackable className="session">
+      {head}
+      <Grid.Row className="titleRow">
+        {/* HACK delete slice and get correct user ID */}
+        <Grid.Column width="10"><strong>{id.slice(0, -15)}</strong> {name}</Grid.Column>
```

Comment:
Prefer doing stuff like this on the outside of the component (pass the slice as a prop). This makes it obvious that a slice was used and the component doesn't interfere with the input in bad ways.